template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl

```diff
@@ -44,21 +44,17 @@ to address any test failures.
 {{ if (eq .input_include_feature_store `yes`) }}
 The integration test runs the feature engineering and model training notebooks as a multi-task Databricks Job in the staging workspace.
 It reads input data, performs feature transforms, and writes outputs to Feature Store tables in the staging workspace. 
-The model training notebook uses these Feature Store tables as inputs to train, validate and register a new model version in the model registry. 
+The model training notebook uses these Feature Store tables as inputs to train, validate and register a new model version in UC. 
```

Comment:
Let's guard this with an if statement only if the models in UC option is selected